databases/sqlitebrowser/Portfile

```diff
@@ -56,9 +58,8 @@ if {[info procs cmake.save_configure_cmd] ne ""} {
 }
 
 configure.args-append \
-                    -DUSE_QT5=ON \
                     -DQSCINTILLA_INCLUDE_DIR=${qt_includes_dir} \
-                    -DQSCINTILLA_LIB_DIR=${qt_frameworks_dir}
+                    -DQT_LIBRARY_DIR=${qt_frameworks_dir}
```

Comment:
You'd have to do the same for the Linux hunk.

Reply:
Thanks. Done.